src/Nri/Ui/SegmentedControl/V7.elm

```diff
@@ -142,17 +175,23 @@ viewTab maybeToUrl config option =
                             :: attrs
                         )
                         children
+
+        ariaCurrent =
+            if forPage then
+                Aria.currentPage
+
+            else
+                Widget.selected True
     in
     element
         (List.concat
             [ [ Attr.id idValue
               , Role.tab
-              , Aria.controls (panelIdFor option)
```

Comment:
Turns out Aria.controls doesn't do much and we get errors when we have controls for an id that doesn't exist on the page (https://github.com/reach/reach-ui/issues/4). Additionally, we only show the id of the selected content (https://github.com/NoRedInk/noredink-ui/pull/363/files#diff-ae19bbc48c636566ee966b8343810172L100) so the other tab options throw axe errors. It was best to delete